native/ipc.js

```diff
@@ -9,6 +10,8 @@ export const OPEN_PATH = "IPC_OPEN_PATH";
 //
 // The workaround is to use the electron native open dialog. As a bonus we
 // can configure it to allow users to create new directories.
+export const getVersion = () => window.electron.ipcRenderer.invoke(GET_VERSION);
```

Comment:
It looks like this function got in between the comment above that belongs to the function below.

Reply:
Good catch!

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/901/commits/e85f82566712684a9e0f32e5e16c531a48c11192